core/src/main/java/org/testcontainers/containers/wait/internal/InternalCommandPortListeningCheck.java

```diff
@@ -29,7 +29,7 @@ public Boolean call() {
 
     private void tryPort(Integer internalPort) {
         String[][] commands = {
-                {"/bin/sh", "-c", format("cat /proc/net/tcp | awk '{print $2}' | grep :%x && echo %s", internalPort, SUCCESS_MARKER)},
+                {"/bin/sh", "-c", format("cat /proc/net/tcp | awk '{print $2}' | grep :%X && echo %s", internalPort, SUCCESS_MARKER)},
```

Comment:
%x produces lower-case hexadecimal numbers, /proc/net/tcp has uppercase references. This only shows itself with ports that actually contain letters, of course. Hence the configuration change for the nginx instance to run on 8080, which is '1F90' in hex (as opposed to '1f90')...

Reply:
whoa! Nice one, thanks @dnno. We should probably change grep to `-i` to completely ignore the case in case of the case differences. WDYT?

Reply:
Right, that sounds reasonable. Will change that!